index.html

```diff
@@ -1291,16 +1236,6 @@ <h4>
         to the web page using the Web NFC API.
       </p>
     </section>
-    <section> <h4>Record URL host and path when pushing</h4>
```

Comment:
what about not completely removing this, but transforming to the `id` parlance?

Reply:
Can you copy out the text and put into an issue...

It probably makes sense to write by default (but that then will require more spec changes), but it might also be an issue for some legacy systems so we might want a way to opt out.

Reply:
but now we lose this content and we should not... it would belong to this change round

Reply:
Do you mind adding that then? I have to pick up my daughter soon :-)

Reply:
well it's there, what we should do is not removing it... but reformulate with `id`
no need to do it now, but you have the right context. It should really be contained in this PR IMHO.

Reply:
OK readded